.github/workflows/go-production.yml

```diff
@@ -0,0 +1,210 @@
+        shell: bash
+        run: |
+          echo "BRANCH_NAME=$(echo ${GITHUB_REF#refs/heads/} | tr / -)" >> $GITHUB_ENV
+
+      - name: Get branch name for pull request
+        if: github.event_name == 'pull_request'
+        shell: bash
+        run: |
+          echo "BRANCH_NAME=$(echo ${GITHUB_BASE_REF} | tr / -)" >> $GITHUB_ENV
+
+      - name: Debug branch name
+        run: echo ${{ env.BRANCH_NAME }}
+
+      - name: Generate ref tag (production)
+        if: ${{ env.BRANCH_NAME == 'production' }}
+        run: |
+          echo "REF_TAG=production" >> $GITHUB_ENV
+
+      - name: Append sufix on VERSION file for production build
+        run: |
+          echo "`cat ${{github.workspace}}/VERSION`-${{ env.REF_TAG }}" > VERSION
+
+      - name: Get VERSION
+        run: |
+          echo "VERSION=`cat ${{github.workspace}}/VERSION`" >> $GITHUB_ENV
+
+      - name: Debug VERSION
+        run: echo ${{ env.VERSION }}
+
+      - name: Build services (go build only)
+        run: make services
+
+      - name: Build orb agent (go build only)
+        if: steps.filter.outputs.agent == 'true'
+        run: make agent_bin
+
+      - name: Go unit tests
+        run: make test
+
+      - name: Build UI
+        if: steps.filter.outputs.ui == 'true'
+        run: make ui
+
+  package:
+    needs: build
+    runs-on: ubuntu-latest
+    # if this is a push into one of our main branches (rather than just a pull request), we will also package
+    if: github.event_name != 'pull_request'
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: dorny/paths-filter@v2
+        id: filter
+        with:
+          filters: |
+            agent:
+              - 'agent/**'
+              - 'cmd/agent/**'
+            fleet:
+              - 'fleet/**'
+              - 'cmd/fleet/**'
+            policies:
+              - 'policies/**'
+              - 'cmd/policies/**'
+            sinks:
+              - 'sinks/**'
+              - 'cmd/sinks/**'
+            sinker:
+              - 'sinker/**'
+              - 'cmd/sinker/**'
+            ui:
+              - 'ui/**'
+
+      - name: Get short commit hash to a variable
+        id: commit_hash
+        run: |
+          echo "::set-output name=sha_short::$(git rev-parse --short HEAD)"
+
+      - name: Get version to a variable
+        id: version
+        run: |
+          echo "::set-output name=version::$(cat VERSION)"
+
+      - name: Commit orb-ui-live on orb-live repo production
+        if: env.BRANCH_NAME == 'production'
+        run: |
+          git config --global user.email "${{secrets.GH_ORB_EMAIL}}"
+          git config --global user.name "${{secrets.GH_ORB_USER}}"
+          git config --global credential.helper cache
+          git clone -b main https://${{secrets.GH_ORB_USER}}:${{secrets.GH_ORB_ACCESS_TOKEN}}@github.com/ns1labs/orb-live.git          
+          cd orb-live
+          cp -rf ../ui .
+          cp -rf ../VERSION .
```

Comment:
again, this won't have `-production` suffix?

Reply:
this "package" job has a clean checkout though, I believe - it doesn't reuse the "build" job where you added the suffix... maybe you should double check that

Reply:
Until I've checked I append suffix on the second step of **build** stage (first stage), and it will follow VERSION file entire pipeline stages (build, package), thus im commiting on that repository (orb-live) a VERSION file with suffix.